e2e/tests/mobile/guest-shopper.spec.js

```diff
@@ -34,24 +34,31 @@ test("Guest shopper can checkout items as guest", async ({ page }) => {
 
   await clothingNav.click();
 
-  await page.getByRole("link", { name: "Tops" }).click();
+  const topsLink = page.getByRole("link", { name: "Tops" })
```

Comment:
```suggestion
  const topsLink = page.getByLabel('Womens').getByRole("link", { name: "Tops" })
```

Resolves ` Error: locator.waitFor: Error: strict mode violation: getByRole('link', { name: 'Tops' }) resolved to 2 elements`